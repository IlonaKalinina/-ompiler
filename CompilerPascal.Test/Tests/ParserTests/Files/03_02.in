program Test_03_02;
var
i : integer = 1;
begin
if (i > -5) or (i < 5) then write('yes');
end.