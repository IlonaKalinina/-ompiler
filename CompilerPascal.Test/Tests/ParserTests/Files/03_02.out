program Test_03_02
├───var
│    └─── :
│         ├─── i
│         ├─── integer
│         └─── =
│              └─── 1
│    
begin
├─── if
│    ├─── or
│    │    ├─── >
│    │    │    ├─── i
│    │    │    └─── -
│    │    │         └─── 5
│    │    └─── <
│    │         ├─── i
│    │         └─── 5
│    └─── write
│         └─── yes
└─── end
